openmdao/core/system.py

```diff
@@ -163,7 +163,7 @@ class System(object):
         Array of local sizes of this system's allprocs variables.
         The array has size nproc x num_var where nproc is the number of processors
         owned by this system and num_var is the number of allprocs variables.
-    _owned_var_sizes : ndarray
+    _owned_sizes : ndarray
         Array of local sizes for 'owned' or distributed vars only.
```

Comment:
You should merge out. I made these same changes on a branch that is on master.

Reply:
done